prow/kube/prowjob.go

```diff
@@ -67,3 +73,17 @@ const (
 	// carries the Optional from a Presubmit job.
 	IsOptionalLabel = "prow.k8s.io/is-optional"
 )
+
+var (
+	ComponentSpecifiedAnnotationsAndLabels = sets.NewString(
+		// Labels
+		client.GerritRevision,
```

Comment:
this would cause circular dependency, as the `gerrit/client` package imports `kube` package. I would suggest either:

a). Define all these Gerrit specific labels in this file
or
b). Define ComponentSpecifiedAnnotationsAndLabels at wherever it will be used